pkg/error/__snapshots__/error_test.snap

```diff
@@ -0,0 +1,22 @@
+
+[TestPrintErrorCount/nonverbose-ten - 1]
+[31;1m
+10 errors found[33;0m
+
+---
+
+[TestPrintErrorCount/nonverbose-zero - 1]
+
+---
+
+[TestPrintErrorCount/verbose-ten - 1]
+[31;1m
+10 errors found[33;0m
```

Comment:
Could the --no-color mode be tested too?

Nothing test it for now and here could be a good place

Reply:
I think this should be it's own test in the logger package

Reply:
We actually have testing of the uncolorized mode:
https://github.com/editorconfig-checker/editorconfig-checker/blob/main/pkg/logger/logger_test.go#L56-L59
https://github.com/editorconfig-checker/editorconfig-checker/blob/main/pkg/logger/logger_test.go#L76-L79